TriblerGUI/defs.py

```diff
@@ -115,7 +171,7 @@
 
 # This list of content categories is used in drop-down menu when editing a personal channel
 # TODO: build this automatically and/or move it somewhere
-CATEGORY_LIST = [
+CATEGORY_LIST = (
```

Comment:
Why is this a tuple now?

Reply:
Because tuples are immutable, which is safer and faster. Should have been like this from the beginning.

Reply:
Agreed, we should change the other definitions to tuples also (at least for most of the lists in `defs.py`). Let's not make these changes in this PR.